### PR TITLE
repair: Fix finished ranges metrics for removenode

### LINKED ISSUE #21174: node_ops_finished_percentage metric for removenode with RBNO never reaching 1, even though removenode finishes
Reproduces only with RBNO.
Doesn't reproduce if I disable RBNO (i.e. use "old streaming") with `enable_repair_based_node_ops: False`.

Checked on latest `master` (df6991edd38d97fff91a1bcdb0145ec68d21c590), 2024.1 (scylladb/scylla-enterprise@6140bb5b2d0a4e0dc425148f6a42dd7fad74f637). Tried to check on 2023.1, but got stuck due to https://github.com/scylladb/scylla-enterprise/issues/4838.

But I'm guessing this problem existed ever since RBNO was introduced.

To reproduce:
- start 3 node cluster
- stop one node
- `nodetool removenode`
- after it finishes, check metrics
- observe that `scylla_node_ops_finished_percentage{ops="removenode",shard="X"}` is not 1 for some shard; in my repro I used 3 shards on each node, and shard="2" and shard="1" had the problem

The metric stays that way.

```
$ curl -s 127.0.0.10:9180/metrics | grep node_ops
# HELP scylla_node_ops_finished_percentage Finished percentage of node operation on this shard
# TYPE scylla_node_ops_finished_percentage gauge
scylla_node_ops_finished_percentage{ops="bootstrap",shard="0"} 1.000000
scylla_node_ops_finished_percentage{ops="decommission",shard="0"} 1.000000
scylla_node_ops_finished_percentage{ops="rebuild",shard="0"} 1.000000
scylla_node_ops_finished_percentage{ops="removenode",shard="0"} 0.679269
scylla_node_ops_finished_percentage{ops="repair",shard="0"} 1.000000
scylla_node_ops_finished_percentage{ops="replace",shard="0"} 1.000000
scylla_node_ops_finished_percentage{ops="bootstrap",shard="1"} 1.000000
scylla_node_ops_finished_percentage{ops="decommission",shard="1"} 1.000000
scylla_node_ops_finished_percentage{ops="rebuild",shard="1"} 1.000000
scylla_node_ops_finished_percentage{ops="removenode",shard="1"} 0.679269
scylla_node_ops_finished_percentage{ops="repair",shard="1"} 1.000000
scylla_node_ops_finished_percentage{ops="replace",shard="1"} 1.000000
scylla_node_ops_finished_percentage{ops="bootstrap",shard="2"} 1.000000
scylla_node_ops_finished_percentage{ops="decommission",shard="2"} 1.000000
scylla_node_ops_finished_percentage{ops="rebuild",shard="2"} 1.000000
scylla_node_ops_finished_percentage{ops="removenode",shard="2"} 0.679269
scylla_node_ops_finished_percentage{ops="repair",shard="2"} 1.000000
scylla_node_ops_finished_percentage{ops="replace",shard="2"} 1.000000
```

This has consequences for our monitoring: see my observation in https://github.com/scylladb/scylladb/issues/21120#issuecomment-2419794173 and @amnonh response https://github.com/scylladb/scylladb/issues/21120#issuecomment-2421427328

Apparently our monitoring stack is using this metric to determine when topology operations are running.

cc @asias

### PR DESCRIPTION
**Pull request by asias**
The skipped ranges should be multiplied by the number of tables

Otherwise the finished ranges ratio will not reach 100%.

Fixes #21174

### CODE REVIEW

File: test/topology_custom/test_node_ops_metrics.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import asyncio
+import pytest
+import logging
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_bootstrap_removenode_metrics(manager):
```
Comment (denesb) [inline test/topology_custom/test_node_ops_metrics.py:8]: Does this test reproduce the problem?

**Review by denesb — APPROVED** [review]

File: test/topology_custom/test_node_ops_metrics.py (hunk 1 above)
Comment (asias) [reply test/topology_custom/test_node_ops_metrics.py:8]: Yes

File: test/topology_custom/test_node_ops_metrics.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import asyncio
```
Comment (avikivity) [inline test/topology_custom/test_node_ops_metrics.py:1]: Missing license/copyright blurb

Comment (asias) [reply test/topology_custom/test_node_ops_metrics.py:6]: OK. Fixed and pushed a new version.

**Comment by asias** [comment]
Changed in 9868ccbac02099f74b6b16f9ea811faf46e562ee:
- Added copyright info for test_node_ops_metrics.py

**Pull request merged**